LL-2276 ~ LL-2304 ~ LL-2328

LFBarreto opened the pull request:
(TRX): add claim rewards

includes #1189  maybe merge it first if everything's good

### Type

Feature

### Context

L-2276 ~ LL-2304

LL-2328

### Parts of the app affected / Test plan

Trx accounts with claimable rewards.

gre commented on src/screens/FreezeFunds/04-ValidationSuccess.js:
```diff
@@ -39,7 +35,7 @@ const useTimer = (timer: number) => {
 
 type Props = {
-  account: ?(TokenAccount | Account),
+  account: ?Account,
```
I think this should either be `AccountLike` OR you should drop parentAccount concept ?

gre commented on src/screens/FreezeFunds/04-ValidationError.js:
```diff
@@ -15,7 +15,7 @@ import { urls } from "../../config/urls";
 type Props = {
-  account: AccountLike,
+  account: Account,
   parentAccount: ?Account,
```
what is parentAccount for if account is already a Account ?

gre commented on src/screens/UnfreezeFunds/01-Amount.js:
```diff
@@ -0,0 +1,334 @@
+    bridgePending,
+    bridgeError,
+  } = useBridgeTransaction(() => {
+    const { tronResources: { frozen: { bandwidth } = {} } = {} } = account;
```
```js
const { tronResources } = account;
invariant(tronResources, "tron resources expected");
const { frozen: { bandwidth } } = tronResources;
```

all the things please =D

- this is more readable.
- this give better typing and avoid you do to all the `bandwidth && bandwidth.amount ` & co..

gre commented:
ok in fact there is a conflict; i'll let you resolve it :)

LFBarreto commented:
@gre i changed all the concerned files :+1:  :
- i removed the parentAccount since its not necessary
- i removed unused translate connect i left from copy pasta
- cleaned up import orders and some dupe variables i found
Should i close the other #1189 since this one includes it and the fixes?
